judge/views/widgets.py

```diff
@@ -100,6 +100,17 @@ def pdf_statement_uploader(statement):
     return urljoin(url_base, name)
 
 
+def submission_uploader(submission_file):
+    ext = os.path.splitext(submission_file.name)[1]
+    name = str(uuid.uuid4()) + ext
+    default_storage.save(os.path.join(settings.SUBMISSION_FILE_UPLOAD_MEDIA_DIR, name), submission_file)
+    url_base = getattr(settings, 'SUBMISSION_FILE_UPLOAD_URL_PREFIX',
```

Comment:
This URL doesn't come with a domain, so we need to join it with the domain before send to the judge